migrate.ts

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ts-node-script
+
+import { Sequelize } from 'sequelize';
+// tslint:disable-next-line: no-implicit-dependencies
+import { Umzug, SequelizeStorage, Migration } from 'umzug';
+import { sequelizeConfig } from './src/db';
+import Path from 'path';
+
+const sequelize = new Sequelize({ ...sequelizeConfig, logging: undefined });
+
+const umzug = new Umzug({
+  logging: console.info,
+  migrations: {
+    path: './src/migrations',
+    pattern: /\.ts$/,
+    params: [sequelize.getQueryInterface(), Sequelize],
+    nameFormatter(path) {
+      // ignore file extension to make it compatible with older .js migrations
+      return Path.basename(path).replace(Path.extname(path), '');
```

Comment:
```suggestion
      return Path.basename(path, Path.extname(path));
```

https://nodejs.org/api/path.html#path_path_basename_path_ext
seems to be a safer alternative